src/devtools/client/debugger/src/reducers/pause.ts

```diff
@@ -315,18 +325,15 @@ const pauseSlice = createSlice({
     },
     resumed(state) {
       Object.assign(state, resumedPauseState);
-      state.threadcx.pauseCounter++;
       state.threadcx.isPaused = false;
     },
   },
   extraReducers: builder => {
     builder
       .addCase(executeCommandOperation.pending, (state, action) => {
         const { command } = action.meta.arg;
-
         state.command = command;
         state.lastCommand = command;
-        state.threadcx.pauseCounter++;
```

Comment:
this might be the part i'm least sure of. I removed a bunch of these incrementers `state.threadcx.pauseCounter` so that we only bumped it when we truly paused. I think that is safe, but it would be worth thinking about.